API/api/services/slot_service.py

```
@@ -0,0 +1,60 @@
+        
+        plage = data["plage"]
+        event = plage.event
+
+        plage_start = make_aware(datetime.combine(event.event_date, plage.start_time))
+        plage_end = make_aware(datetime.combine(event.event_date, plage.end_time))
+
+        if data["start_at"] < plage_start or data["end_at"] > plage_end:
+            raise ValidationError({
+                "non_field_errors": (
+                    "Le créneau doit être compris dans la plage "
+                    f"({plage_start} – {plage_end})."
```

Ruff (0.15.10)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)